packages/api/src/external/commonwell/organization.ts

```diff
@@ -100,110 +120,129 @@ export async function get(orgOid: string): Promise<CWOrganization | undefined> {
   } catch (error: any) {
+    const cwRef = commonWell.lastReferenceHeader;
     const extra = {
-      orgId: org.id,
-      orgOID: org.oid,
-      cwReference: commonWell.lastReferenceHeader,
+      orgOid: org.oid,
+      cwReference: cwRef,
       context: `cw.org.update`,
+      commonwellOrg,
       error,
     };
-    // Try to create the org if it doesn't exist
     if (error.response?.status === 404) {
-      capture.message("Got 404 when updating Org @ CW, creating it", { extra });
-      return create(org);
+      capture.message("Got 404 while updating Org @ CW, creating it", { extra });
+      await create(cxId, org, isObo);
     }
-    // General error handling
-    const msg = `Failure updating Org @ CW`;
-    log(msg, error);
-    capture.message(msg, { extra: { ...extra, payload: cwOrg }, level: "error" });
+    const msg = `Failure while updating org @ CW`;
+    log(`${msg}. Org OID: ${org.oid}. Cause: ${errorToString(error)}. CW Reference: ${cwRef}`);
+    capture.error(msg, { extra });
     throw error;
   }
 }
 
-export async function initCQOrgIncludeList(orgOID: string): Promise<void> {
-  const { log } = out(`initCQOrgIncludeList - orgOID ${orgOID}`);
+export async function initCQOrgIncludeList(orgOid: string): Promise<void> {
+  const { log } = out(`CW initCQOrgIncludeList - CW Org OID ${orgOid}`);
   try {
     const managementApi = makeCommonWellManagementAPI();
     if (!managementApi) {
-      log(`Not linking org ${orgOID} to CQ Bridge b/c no managementAPI is available`);
+      log(`Not linking org ${orgOid} to CQ Bridge b/c no managementAPI is available`);
       return;
     }
     const highPrioOrgs = getOrgsByPrio().high;
     const cqOrgIds = highPrioOrgs.map(o => o.id);
     const cqOrgIdsLimited =
       cqOrgIds.length > MAX_HIGH_PRIO_ORGS ? cqOrgIds.slice(0, MAX_HIGH_PRIO_ORGS) : cqOrgIds;
-    log(`Updating CQ include list for org ${orgOID} with ${cqOrgIdsLimited.length} high prio orgs`);
-    await managementApi.updateIncludeList({ oid: orgOID, careQualityOrgIds: cqOrgIdsLimited });
+    log(`Updating CQ include list for org ${orgOid} with ${cqOrgIdsLimited.length} high prio orgs`);
+    await managementApi.updateIncludeList({ oid: orgOid, careQualityOrgIds: cqOrgIdsLimited });
   } catch (error) {
-    const extra = { orgOID, context: `initCQOrgIncludeList` };
     const msg = `Error while updating CQ include list`;
-    log(`${msg}. Cause: ${errorToString(error)}`, extra);
-    capture.message(msg, {
-      extra: { ...extra, error },
-      level: "error",
+    log(`${msg}. Cause: ${errorToString(error)}`);
+    capture.error(msg, {
+      extra: {
+        orgOid,
+        context: `cw.org.initCQOrgIncludeList`,
+        error,
+      },
     });
+    throw error;
   }
 }
+
+export function parseCWEntry(org: CWSdkOrganization): CWOrganization {
+  const location = (
+    org.locations as [CWSdkOrganizationLocation, ...CWSdkOrganizationLocation[]]
```

Comment:
Why do we need to do `as [CWSdkOrganizationLocation, ...CWSdkOrganizationLocation[]]`

Reply:
You're right, cleaned up the typing (just casted the `org.locations[0]` -- it's a big cleaner. Not sure what to do if really an org has no location, but I'm assuming this is not really possible.